packages/autorest.typescript/src/restLevelClient/transforms/transform.ts

```diff
@@ -4,17 +4,17 @@
 import {
   CodeModel,
   ImplementationLocation,
-  ParameterLocation,
-  Protocol
+  ParameterLocation
 } from "@autorest/codemodel";
 import {
-  ImportKind,
+  initInnerImports,
   RLCModel,
   HelperFunctionDetails,
   UrlInfo,
   ApiVersionInfo,
   extractPathApiVersion,
-  extractDefinedPosition
+  extractDefinedPosition,
+  build3ndPartyImports
```

Comment:
build3rdPartyImports or buildNonbrandingImports

Reply:
The `third party` is mainly for SDK's dependency not for branding or non-branding. I sperated the inner imports which means the imports from the SDK iteself e.g models or parameters ect and third party imports which means the depedency outside of the SDK.

So both the `@azure/core-rest-pipeline` or `@typespec/ts-http-runtime` are third parties.